src/index.html

```diff
@@ -5,8 +5,365 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo"><img src="images/logo.png" alt="/"></a>
+      
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-item" data-qa="nav-hover">
+            <a href="#" class="nav__link">
+              Laptops &amp; Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content">
+      <ul class="content__list">
+        <div class="cards">
+          <container class="product" data-qa="card">
+            <img class="product__image"
+         src="images/imac.jpeg"
+         alt="APPLE A1419 iMac 27">
+
+            <h2 class="product__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
```

Comment:
```suggestion
            <h2 class="product__name">
              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
            </h2>
```
check your code style
fix all cases